### PR TITLE
replace rapidjson

### PR DESCRIPTION
**Pull request by xiepengcheng01**
# Proposed changes

Issue Number: open #7551 

## Problem Summary:

replace rapidjson with simdjson for handle simple json.

## Checklist(Required)

1. Does it affect the original behavior: (Yes/No/I Don't know)
2. Has unit tests been added: (Yes/No/No Need)
3. Has document been added or modified: (Yes/No/No Need)
4. Does it need to update dependencies: (Yes/No)
5. Are there any changes that cannot be rolled back: (Yes/No)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: be/CMakeLists.txt [hunk 1]
```diff
@@ -143,9 +143,6 @@ set_target_properties(gflags PROPERTIES IMPORTED_LOCATION ${THIRDPARTY_DIR}/lib/
 add_library(glog STATIC IMPORTED)
 set_target_properties(glog PROPERTIES IMPORTED_LOCATION ${THIRDPARTY_DIR}/lib/libglog.a)
 
-add_library(backtrace STATIC IMPORTED)
```
Comment (yangzhg) [inline be/CMakeLists.txt:146]: why modify here？ need rebase？

File: be/src/exec/json_scanner.h [hunk 2]
```diff
@@ -147,14 +148,21 @@ class JsonReader {
     void _fill_slot(Tuple* tuple, SlotDescriptor* slot_desc, MemPool* mem_pool,
                     const uint8_t* value, int32_t len);
     Status _parse_json_doc(size_t* size, bool* eof);
+    Status _simdjson_parse_json_doc(size_t* size, bool* eof);
```
Comment (yangzhg) [inline be/src/exec/json_scanner.h:151]: why use _simdjson_parse_json_doc? why not replace _parse_json_doc directly?

File: be/src/exec/json_scanner.cpp [hunk 3]
```diff
@@ -360,6 +365,79 @@ void JsonReader::_close() {
...
+        }
+        return Status::DataQualityError(error_msg.data());
+    }
+    _element = document.value_unsafe();
+
+    if (_element.is_array() && !_strip_outer_array) {
+        fmt::memory_buffer error_msg;
+        fmt::format_to(error_msg, "{}", "JSON data is array-object, `strip_outer_array` must be TRUE.");
+        RETURN_IF_ERROR(_state->append_error_msg_to_file([&]() -> std::string { return _print_json_value(_origin_json_doc); },
+                [&]() -> std::string { return error_msg.data(); }, _scanner_eof));
```
Comment (yangzhg) [inline be/src/exec/json_scanner.cpp:414]: use fmt::to_string(error_msg), not use error_msg.data()

File: be/CMakeLists.txt (hunk 1 above)
Comment (xiepengcheng01) [reply be/CMakeLists.txt:146]: yes, need rebase.

**Pull request closed without merging**